### ISSUE TITLE
Can't update a pull request from master branch

### ISSUE THREAD

**Issue by crossan007** [k0shk0sh/FastHub#1379]
**FastHub Version: 4.6.0**  
**Android Version: 7.1.2 (SDK: 25)**  
**Device Information:**  
- **Manufacturer:** LGE  
- **Brand:** lge  
- **Model:** LG G5  
---

The git hub web ui has a button to update a branch from master when it has an open pull request.  This is often useful for triggering CI testing on a feature branch which is lagging behind master.

**Comment by k0shk0sh** [comment]
Thanks for the request however I don't believe that there is such api for that. 

If I ever found GitHub has such api I'll reopen this ticket.

**Comment by crossan007** [comment]
It looks like this would be possible using the merge endpoint:
> You can perform a merge using the merge endpoint: https://developer.github.com/v3/repos/merging/8

> To match the behavior of the Update branch button, which is to merge master into your branch, you would pass these args:
```
{
 "base": "your_branch",
 "head": "master",
 "commit_message": "Merge master into branch"
}
```

From: https://platform.github.community/t/update-branch-button-equivalent-in-the-api/4260

**Comment by k0shk0sh** [comment]
Makes sense, I'll look into the response later to check if there is any variable that can tell me that the branch needs to be updated. If not I'll just add the update btn anyway.

**Comment by crossan007** [comment]
Thanks!!!!
